docs/openapi/components/schemas/common/TransportEvent.yml

```diff
@@ -0,0 +1,145 @@
+$linkedData:
+  term: TransportEvent
+  '@id': https://w3id.org/traceability#TransportEvent
```

Comment:
```suggestion
  '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#Event
```

Reply:
The world has (more than) plenty of such shipping event schemes already.

Reply:
also, concerned about Buy-Ship-Pay workflow mandates, etc

Reply:
`https://service.unece.org/trade/uncefact/vocabulary/uncefact/#transportEvent`